### PR TITLE
Cumulus-1713 Removing integration tests error message

### PR DESCRIPTION
**Pull request by kkelly51**
Turns off SNS collection reporting for integration tests. Should no longer get error about missing SNS Topic ARN

### CODE REVIEW

File: packages/api/models/collections.js [hunk 1]
```diff
@@ -169,15 +171,17 @@ class Collection extends Manager {
...
-    const record = {
-      event: 'Delete',
-      deletedAt: Date.now(),
-      record: {
-        name,
-        version
-      }
-    };
-    await publishCollectionSnsMessage(record);
+    if (!process.env.noSNS) {
```
Comment (markdboyd) [inline packages/api/models/collections.js]: I have mixed feelings about using env vars specific to testing in our real API package. See suggestion below

File: packages/integration-tests/index.js [hunk 2]
```diff
@@ -256,6 +256,7 @@ function setProcessEnvironment(stackName, bucketName) {
   process.env.CollectionsTable = `${stackName}-CollectionsTable`;
   process.env.ProvidersTable = `${stackName}-ProvidersTable`;
   process.env.RulesTable = `${stackName}-RulesTable`;
+  process.env.noSNS = true;
```
Comment (markdboyd) [inline packages/integration-tests/index.js]: Is there any reason to not go the other way and set the actual env var that will allow the publish requests to the collections topic to succeed? That would also eliminate the error message

```
process.env.collection_info_sns_topic_arn = `${stackName}-collection-information-topic`
```

**Comment by nemreid** [comment]
I think the issue here for integration tests is that we are calling `collectionModel.delete` in `addCollections`, instead of invoking the API lambda to delete the collection instead. That's why the `process.env.collection_sns_topic_arn` variable is unset, and an error is thrown because the collection fails to add.

We could set the env, as Mark suggested, but it would be cleaner to refactor so that the API is called instead of the models in our tests. This would also remove the error because the API has the env var set.

I understand if that is considered out of scope, though, as it may be a fair bit of work.

**Comment by yjpa7145** [comment]
> I think the issue here for integration tests is that we are calling `collectionModel.delete` in `addCollections`, instead of invoking the API lambda to delete the collection instead. That's why the `process.env.collection_sns_topic_arn` variable is unset, and the warning is thrown because the collection delete call can't publish.
> 
> We could set the env, as Mark suggested, but it would be cleaner to refactor so that the API is called instead of the models in our tests. This would also remove the error because the API has the env var set.
> 
> I understand if that is considered out of scope, though, as it may be a fair bit of work.

It's actually a pretty small change: https://github.com/nasa/cumulus/pull/1494

**Comment by kkelly51** [comment]
Closing because better fix in PR [#1494](https://github.com/nasa/cumulus/pull/1494)

**Pull request closed without merging**